base_phone/models/phone_common.py

```diff
@@ -5,11 +5,15 @@
 from openerp import models, api
 from openerp.addons.base_phone.fields import Phone
```

Comment:
Actually can you link to the fail in Travis? I think that's something manipulated due to PIP